service-worker/worker/src/e2e/simple/sanity.e2e.ts

```diff
@@ -0,0 +1,98 @@
+import {create, Server} from '../../testing/harness/server/server';
+import {HarnessPageObject} from '../page-object';
+declare var browser;
+declare var element;
+declare var by;
+
+
+let server: Server;
+let po: HarnessPageObject;
+
+const SIMPLE_MANIFEST = {
+  group: {
+    default: {
+      version: 'test',
+      url: {
+        '/hello.txt': {}
+      }
+    }
+  }
+};
+
+beforeAll(done => {
+  create(8080, 'dist/harness/client').then(s => {
```

Comment:
If this setup can be shared for all tests without leaking state, can we move it into protractor.config?